src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -24,6 +24,11 @@
 * Fixed output container in re-registration scenario for Register-AzRecoveryServicesBackupContainer cmdlet.
 * Added support for Enabling/Disabling the azure monitor and classic alerts for recovery services vault.
 
+## Version 5.4.2
```

Comment:
Please put your changelog under the "Upcoming Release" title, and do not add extra titles.